docs-kr/network/edges.html

```diff
@@ -886,6 +890,27 @@ <h3>옵션</h3>
             <td><code>false</code></td>
             <td>to Node와 from Node가 동일한 경우, 원이 그려집니다. 이것은 그 원의 반지름입니다.</td>
         </tr>
+        <tr class='toggle collapsible' onclick="toggleTable('optionTable','selfReference', this);">
+            <td><span parent="arrows" class="right-caret"></span> selfReference</td>
+            <td>Object</td>
+            <td><code>Object</code></td>
+            <td>TODO
+            </td>
+        </tr>
+        <tr parent="selfReference" class="hidden">
+            <td class="indent">size</td>
+            <td>Number</td>
+            <td><code>false</code></td>
```

Comment:
The default value for datatype number cannot be false. In the code it was and is 20.